docs/guides/signature-verification.md

```diff
@@ -0,0 +1,135 @@
+# Dasharo release signature verification
+
+Each published Dasharo release is signed with a signing key corresponding to
+given platform and versions. The key infrastructure is stored in
+[3mdeb-secpack](https://github.com/3mdeb/3mdeb-secpack).
+
+To verify the integrity of the binaries published in release notes on this
+site, please follow the instructions below:
+
+1. Import necessary keys to your keyring:
+
+    ```bash
+    wget https://raw.githubusercontent.com/3mdeb/3mdeb-secpack/master/keys/master-key/3mdeb-master-key.asc \
+        -O - | gpg --import -
+
+    wget https://raw.githubusercontent.com/3mdeb/3mdeb-secpack/master/dasharo/3mdeb-dasharo-master-key.asc  \
+        -O - | gpg --import -
+
+    wget <release signing key URL> -O - | gpg --import -
+    ```
+
+    - `<release signing key URL>` is provided in the release notes
+
+2. CHeck the signatures on the keys:
+
+    ```bash
+    gpg --check-signatures "Dasharo" "3mdeb"
+    ```
+
+3. Download the binaries, SHA sums and their signature files
+
+    ```bash
+    wget <binary>
+    wget <binary>.sha256
+    wget <binary>.sha256.sig
+    ```
```

Comment:
please split for copy pastability or maybe use something like this:

```bash
export BINARY_URL=https://......
```

```bash
wget ${BINARY_URL}
```

```bash
wget ${BINARY_URL}.sha256
```

```bash
wget ${BINARY_URL}.sha256.sig
```

Reply:
@pietrushnic 3499a51c4842c4dddad45a977b376523aa26af12

Reply:
Also change to be more explicit what the binary means: 36ae1fa300d1d09a2be1eda7a146e7dfff436888

Reply:
@miczyg1 is better but not ideal, why you don't want to use a shell variable to not repeat everything?

Reply:
@pietrushnic because it is one more command to paste making it unnecessarily longer. Then one would also have to unexport it after verification. And still after the files are downloaded, the verification requires the local filename, so yet another variable and even more commands. For beginners the more steps the worse.

Reply:
> because it is one more command to paste making it unnecessarily longer.

We add one more command and make the other copy pastable. Without export, we have typing instead of copy-pasting. Typing is more error-prone IMHO. TBH we could probably have autogenerated curl command that does everything in oneliner.

> Then one would also have to unexport it after verification.

I don't see it as an issue. I never export, rather close given terminal.

> And still after the files are downloaded, the verification requires the local filename, so yet another variable and even more commands.

Really?

```shell
export BIN_URL=https://3mdeb.com/open-source-firmware/Dasharo/msi_ms7d25/v1.1.1/msi_ms7d25_v1.1.1_ddr4.rom
wget ${BIN_URL} ${BIN_URL}.sha256 ${BIN_URL}.sha256.sig
gpg -v --verify `basename $BIN_URL`.sha256.sig `basename $BIN_URL`.sha256
sha256sum -c `basename $BIN_URL`.sha256
```

Reply:
@pietrushnic applied: 500e858594867aaf873aec5dacfcdd8eb4bca587